### PR TITLE
Fixed WaPo issue + added lesechos.fr, harpers.org & liberation.fr

### PR DESCRIPTION
**Pull request by iscreamcoke**
Fix WashingtonPost issue + add liberation.fr & les echos.fr

### CODE REVIEW

**Comment by iscreamcoke** [comment]
Updated to add Harper's Magazine

**Comment by iamadamdev** [comment]
Please also update the version number in popup, manifest and update sites in README.md

File: background.js [hunk 1]
```diff
@@ -89,8 +91,11 @@ const remove_cookies = [
 'chicagobusiness.com',
 'theadvocate.com.au',
 'examiner.com.au',
+'lesechos.fr',
+'liberation.fr',
 'hbr.org',
 'medium.com',
+'washingtonpost.com',
```
Comment (iamadamdev) [inline background.js]: This is going to put wapo in an infinite loop for EU users. You can't just remove all the cookies cause they need to accept a GDPR cookie.

Comment (iscreamcoke) [reply background.js]: I'm a EU user and this works for me. I'm not in a loop (only see GDPR at the first visit) but hit paywall count without this modification. See: https://i.imgur.com/3uDhA4k.jpg

Comment (iscreamcoke) [reply background.js]: Waiting for your decision before pulling a PR for the other files

Comment (iamadamdev) [reply background.js]: If it works for you in the EU then I trust you, go ahead and make the changes.
Also can you move Les Echos to after Le Temps

Comment (iscreamcoke) [reply background.js]: I've done all the PRs, should be good now. Best

**Pull request merged**